CustomSecuritySample/AuthenticationExtension.cs

```diff
@@ -0,0 +1,173 @@
+#region Copyright � Microsoft Corporation. All rights reserved.
```

Comment:
Please update the headers with
// Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
// Licensed under the MIT License (MIT)